pkg/eventprocessor/impl/eventprocessor_replayhistory_test.go

```diff
@@ -36,12 +36,12 @@ func TestReplayProductionHistory(t *testing.T) {
 
 	expectedStateHashes := map[tableland.ChainID]string{
 		1:      "ce4f083e256d3458a329b6cd1ba7d8e93d9703b3",
-		5:      "9ab5a92ac961d39ee07dd1e59518c527b39719c1",
-		10:     "3d907df6936eea9ab046d89c80b995d5db26e6ac",
+		5:      "b42740b92e40d616061a88872e19616c05469750",
```

Comment:
i'm not really sure why a change in the state happened. maybe an `INSERT` with `SELECT` query was accepted?